proto/neutron/cron/v1/schedule.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package neutron.cron.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/neutron-org/neutron/v4/x/cron/types/v1";
+
+message Schedule {
+  // Name of schedule
+  string name = 1;
+  // Period in blocks
+  uint64 period = 2;
+  // Msgs that will be executed every period amount of time
+  repeated MsgExecuteContract msgs = 3 [(gogoproto.nullable) = false];
```

Comment:
> every period amount of time

that's confusing. could you please clarify/rephrase that it's about blocks gap between sequential calls? here and in the current `schedule.proto`

Reply:
I've tried to rephrase.